auth_otp/views.py

```
@@ -0,0 +1,357 @@
+"""
+ویوهای سیستم احراز هویت OTP
+OTP Authentication System Views
+"""
+
+from rest_framework import status
+from rest_framework.decorators import api_view, permission_classes
+from rest_framework.permissions import AllowAny, IsAuthenticated
+from rest_framework.response import Response
+from django.utils import timezone
```

Ruff (0.12.2)

`django.utils.timezone` imported but unused

Remove unused import: `django.utils.timezone`

(F401)